OpenAI.Playground/OpenAI.Playground.csproj

```diff
@@ -19,15 +21,25 @@
 		</Content>
 	</ItemGroup>
 
-	<ItemGroup>
+	<ItemGroup Condition="'$(TargetFramework)' != 'netstandard2.0'">
 		<PackageReference Include="LaserCatEyes.HttpClientListener" Version="6.0.0" />
+	</ItemGroup>
+		
+	<ItemGroup>
 		<PackageReference Include="Microsoft.Extensions.Configuration" Version="6.0.1" />
 		<PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="6.0.0" />
 		<PackageReference Include="Microsoft.Extensions.Configuration.CommandLine" Version="6.0.0" />
 		<PackageReference Include="Microsoft.Extensions.Configuration.UserSecrets" Version="6.0.1" />
 		<PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="6.0.1" />
 		<PackageReference Include="Microsoft.Extensions.Options" Version="6.0.0" />
 		<PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="6.0.0" />
+		<PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="6.0.0" />
```

Comment:
why do we need this package?

Reply:
This package is referenced by other packages. eg, Microsoft.Extensions.DependencyInjection/6.0.1, Microsoft.Extensions.Primitives/6.0.0, System.Memory/4.5.4. I manually included the PackageReference line to try to resolve some dependencies but I think it is possible to remove it and have Nuget auto-resolve package dependencies and automatically include it.